camayoc/tests/qcs/cli/test_credentials.py

```diff
@@ -12,7 +12,7 @@
 import random
 from io import BytesIO
 from pathlib import Path
-
+import json
```

Comment:
We don't have any automated check for this but we try and group the imports in three sections:

```
# standard library imports
# standard library "from ... import ..." imports

# third party library imports
# third party library "from ... import ..." imports

# camayoc library imports
# camayoc "from ... import ..." imports
```

So please pop `import json` up a few lines